pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/IngestionDelayTracker.java

```diff
@@ -150,6 +167,19 @@ public IngestionDelayTracker(ServerMetrics serverMetrics, String tableNameWithTy
     _realTimeTableDataManager = realtimeTableDataManager;
     _clock = Clock.systemUTC();
     _isServerReadyToServeQueries = isServerReadyToServeQueries;
+
+    if (realtimeTableDataManager.getInstanceDataManagerConfig() != null
+        && realtimeTableDataManager.getInstanceDataManagerConfig().getConfig() != null) {
+      PinotConfiguration pinotConfiguration = realtimeTableDataManager.getInstanceDataManagerConfig().getConfig();
+      _enableOffsetLagMetric =
+          pinotConfiguration.getProperty("offset.lag.tracking.enable", DEFAULT_ENABLE_OFFSET_LAG_METRIC);
+      _offsetLagUpdateIntervalMs =
+          pinotConfiguration.getProperty("offset.lag.tracking.update.interval", DEFAULT_OFFSET_LAG_UPDATE_INTERVAL_MS);
```

Comment:
Shouldn't `"offset.lag.tracking.enable"` and `"offset.lag.tracking.update.interval"` be named constants?

Reply:
Missed this, declared them as constants